### DISCUSSION TITLE
Livewire can't handle script tag templating inside loops

### DISCUSSION THREAD

**Discussion by reneroboter** [livewire/livewire#4935] [7. Bugs (archived)]
Using script tag with type `text/template` currently not working with Livewire. 

Imagine you have the default counter example and extend it by loop that run until the current count. After the first request Livewire runs into a `Uncaught (in promise) SyntaxError: Unexpected token '<'`. 

This happen because at [node_initalizer](https://github.com/livewire/livewire/blob/master/js/node_initializer.js#L11) Livewire check if the current node is a script tag and throw the current node into JavaScripts `eval` function.

I would expect that Livewire checks here more if the current node is really JavaScript code :) 

I add a simple example of this bug. 

```blade
<div style="text-align: center">
    <button wire:click="increment">+</button>
    <h1>{{ $count }}</h1>
    @for ($i = 0; $i < $count; $i++)
        <script type="text/template">
            <div>Fancy Livewire stuff! 1</div>
        </script>
    @endfor
</div>
```

My current solution is to use template tags instead of script tags of type `text/template`. However I'm not sure if there where other situation where Livewire crash of this condition. What do you think? Does we need here a more explicit condition?

Greeting
René

**Comment by joshhanley** [comment]
@reneroboter I don't think livewire supports `type="text/template"`.

I've never used it myself (as I always use template tags), what do you use it for?

**Reply by reneroboter** [reply]
@joshhanley We used `type="text/template"` for our ads templates. We solved this issue by doing switching to the `template` tag. However I thought livewire should dont run into this issue or what do you think?

**Comment by ruaq** [comment]
Same problem here... I use `<script type="application/ld+json">` for markup in a foreach...
